src/lwc-dev-server/index.ts

```diff
@@ -52,28 +26,19 @@ async function createLWCServerConfig(
   certData?: SSLCertificateData,
   workspace?: Workspace
 ): Promise<ServerConfig> {
-  const sfdxConfig = path.resolve(rootDir, 'sfdx-project.json');
+  const project = await SfProject.resolve();
+  const projectJson = await project.resolveProjectConfig();
+  const packageDirectories = projectJson.packageDirectories;
+  const namespacePaths: string[] = [];
 
-  if (!existsSync(sfdxConfig) || !lstatSync(sfdxConfig).isFile()) {
-    throw new Error(`sfdx-project.json not found in ${rootDir}`);
+  if (!packageDirectories || !Array.isArray(packageDirectories)) {
+    throw new Error('No package directories defined.');
```

Comment:
Give the user a clue about where to define it and a property name or similar. They can search the docs and figure it out. The error message as-is is not actionable.
```suggestion
    throw new Error(`No package directories defined in ${projectJson}.`);
```

Reply:
Changed it to use the API provided by sfdx. It outputs a clean error like this:
```
Error (MissingDefaultPathError): In sfdx-project.json, be sure to specify which package directory (path) is the default. Example: `[{ "path": "packageDirectory1", "default": true }, { "path": "packageDirectory2" }]`
```